internal/testing/jsonpb_test_proto/test_objects.proto

```diff
@@ -1,34 +1,3 @@
-// Go support for Protocol Buffers - Google's data interchange format
```

Comment:
AFAICT, this file did not actually come from Google. I'm not sure how it ended up with this copyright preamble...